demos/common.scss

```diff
@@ -26,7 +26,7 @@ $mdc-theme-secondary: #018786 !default; // baseline teal, 600 tone
 
 // Import button and ripple mixins to apply overrides for dark theme
 // TODO(kfranqueiro): Pending further design discussion around how to manage dark theme
-@import "../packages/mdc-button/mixins";
+@import "../packages/mdc-button/mdc-button";
```

Comment:
why did you make this change?

Reply:
Funny, I had wondered this same thing (and PS this will be removed when we finish removing dark mode anyway)... but I also talked to Andy about this yesterday, mainly about the fact that this PR seems to have a lot of unrelated changes in it, and I think the consensus was he would split this up?

Reply:
A lot of our demo pages have buttons on them. But yes, I will split up this PR